playlist.py

```diff
@@ -0,0 +1,91 @@
+from PyQt5.QtWidgets import QDialog, QFileDialog, QAbstractItemView
+from playlist_ui import Ui_Dialog
+from clip import load_song_from_file, verify_ext
+import json
+from os.path import basename, splitext
+
+
+def getSongs(file_names):
+    r = []
+    for f in file_names:
+        try:
+            r.append(load_song_from_file(f))
```

Comment:
The idea behind this was, that the playlist is used live on stage - since there everything should be loaded as quickly as possible, it made sense to preload all files when using the playlist.
I would argue this would make things overly complicated, but If you don't approve, would you prefer the preloading to be optional?